.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+### Suggested package name(s)
+<!-- Name suggested package(s) -->
+
+### Alternative to
+<!-- Name popular package(s) the suggested one is an alternative to -->
+
+### Why is the suggested package better?
+<!-- Please explain here -->
+
+### Checklist
+- [ ] Package has sufficient overlap in functionality to act as a replacement.
+- [ ] Package is a better alternative to what is already suggested for this category, or the category is new.
+- [ ] Package is not deprecated and actively maintained (the last release was published less than 6 months ago).
```

Comment:
Again, there are stable packages that were released say 1 year ago but are still very popular and useful. A package might be updated very frequently because it has bugs, vs a very high-quality stable package might not have any releases in last N months.

Reply:
You're right.